plugins/wazuh-core/server/services/initialization/types.ts

```diff
@@ -0,0 +1,39 @@
+import { LifecycleService } from '../types';
+
+export interface InitializationTaskDefinition {
+  name: string;
+  run: (ctx: any) => any;
+}
+
+export interface InitializationTaskRunData {
+  name: InitializationTaskDefinition['name'];
+  status: 'not_started' | 'running' | 'finished';
+  result: 'success' | 'fail' | null;
```

Comment:
You could create a constant for each string

Reply:
Added here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7145/commits/409115f401aa99626f39a88bf20fc963b1d61f2b